lib/graphql/directive/deprecated_directive.rb

```diff
@@ -7,5 +7,5 @@
     "suggestion for how to access supported similar data. Formatted "\
     "in [Markdown](https://daringfireball.net/projects/markdown/)."
 
-  argument :reason, !GraphQL::STRING_TYPE, reason_description, default_value: GraphQL::Directive::DEFAULT_DEPRECATION_REASON
+  argument :reason, GraphQL::STRING_TYPE, reason_description, default_value: GraphQL::Directive::DEFAULT_DEPRECATION_REASON
```

Comment:
This PR made me realize I made a mistake in https://github.com/rmosolgo/graphql-ruby/pull/275. `reason` is not required. 😁